Cargo.toml

```diff
@@ -29,6 +29,7 @@ reqwest = "0.11"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 tokio = { version = "1", features = [ "full" ] }
+rand = { version = "0.8", features = [ "std_rng", "getrandom" ], default-features = false }
```

Comment:
`getrandom` is included in the default feature set, so you can just have `rand = "0.8"`, and have the deps sorted(`cargo add -s` will do :)

Reply:
Thanks for the hint, I adapted it accordingly.